src/main/java/seedu/address/model/person/Person.java

```diff
@@ -28,7 +28,7 @@
      * Every field must be present and not null.
      */
     public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, address, tags);
+        requireAllNonNull(name, phone);
```

Comment:
I think you should keep the requireAllNonNull for `Address`, `Email` and `Tags` as well. Based from my understanding of your implementation, you are still creating instance of those classes, except that they hold a placeholder value.
This should help avoid unexpected `NullPointerExceptions` in the future.

Reply:
Resolved by commit 94f73da.